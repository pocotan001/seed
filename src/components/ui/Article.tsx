import margin, { IMarginProps } from "~/components/styles/extends/margin";
import styled from "~/components/styles/themedStyledComponents";

interface IArticleProps extends IMarginProps {
  children: React.ReactNode;
}

const Article = styled<IArticleProps, "article">("article")`
  ${margin};
`;

export default Article;
