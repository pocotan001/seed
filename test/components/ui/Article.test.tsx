import * as React from "react";
import * as renderer from "react-test-renderer";
import Article from "~/components/ui/Article";

describe("<Article>", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Article>alo</Article>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
