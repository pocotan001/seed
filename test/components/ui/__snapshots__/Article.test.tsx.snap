// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Article> should render correctly 1`] = `
<article
  className="Article-b90o5v-0 hLUBpI"
>
  alo
</article>
`;
